app/javascript/components/EventsTimeline.js

```diff
@@ -93,7 +94,7 @@ const EventsTimeline = ({ events, options, isFiltered = false }) => {
         <div ref={eventsContainerEl} onScroll={checkButtons} className="events-container">
           <div className="timeline">
             {currentEvents.map((event) => (
-              <div key={event.title} className="time-point">
+              <div key={event.title} className={cx('time-point', { 'time-point-multiple-events': currentEvents.length > 1 })}>
```

Comment:
maybe with BEM modifier `time-point--multiple-events` I know we are not very consistent with using BEM ;)

but after that CSS could be written like
```
.time-pont {
   ....
   &--multiple-events {
   ...
   }
}
```

Reply:
sure, yeah 👍

Reply:
actually maybe always could be written like `&-multiple-events` haven't checked that.